packages/atlas/src/views/studio/VideoWorkspace/VideoWorkspace.tsx

```diff
@@ -73,183 +49,58 @@ export const VideoWorkspace: React.FC = React.memo(() => {
+          tooltip: {
+            text: 'Drafts system can only store video metadata. Selected files (video, thumbnail) will not be saved as part of the draft.',
+          },
+        }
+      : undefined,
   }
 
   return (
-    <>
-      {isWorkspaceOpen && headTags}
-      <CSSTransition
-        in={isWorkspaceOpen}
-        appear
-        mountOnEnter
-        unmountOnExit
-        timeout={{ enter: 0, exit: parseInt(cVar('animationTimingSlow', true)) }}
-        classNames="video-workspace-drawer"
-      >
-        <DrawerOverlay />
-      </CSSTransition>
-      <CSSTransition
-        in={isWorkspaceOpen}
-        appear
-        mountOnEnter
-        unmountOnExit
-        timeout={{ enter: 0, exit: parseInt(cVar('animationTimingSlow', true)) }}
-        classNames="video-workspace"
-      >
-        <Container role="dialog">
-          <DrawerHeader
-            title={tabData?.title || 'New video'}
-            label={isNftFormOpen ? 'NFT' : editedVideoInfo.isNew || editedVideoInfo.isDraft ? 'New' : 'Edit'}
-            onCloseClick={closeVideoWorkspace}
-          />
-          <ScrollContainer actionBarHeight={actionBarHeight}>
-            <SwitchTransition>
-              <CSSTransition
-                key={String(isNftFormOpen)}
-                classNames={transitions.names.fade}
-                timeout={parseInt(cVar('animationTimingFast', true))}
-              >
-                {!isNftFormOpen ? (
-                  <VideoForm
-                    videoFormDataForNft={videoFormDataForNft}
-                    setFormStatus={setFormStatus}
-                    onSubmit={handleVideoSubmit}
-                    setIsIssuedAsNft={setIsIssuedAsNft}
-                    isIssuedAsNft={isIssuedAsNft}
-                  />
-                ) : (
-                  <NftForm
-                    videoId={editedVideoInfo?.id ?? ''}
-                    setFormStatus={setFormStatus}
-                    onSubmit={handleNftSubmit}
-                    setIsNftFormOpen={setIsNftFormOpen}
-                  />
-                )}
-              </CSSTransition>
-            </SwitchTransition>
-          </ScrollContainer>
-          <VideoWorkspaceActionBar
-            isEdit={isEdit}
-            isIssuedAsNft={isIssuedAsNft}
-            variant={isNftFormOpen ? 'nft' : isEdit ? 'edit' : 'new'}
-            primaryButtonText={formStatus?.actionBarPrimaryText || ''}
-            // form can be submitted if both:
-            // 1. form is valid
-            // 2. the video is a new one OR the form is dirty  (some edit has been made)
-            canSubmit={!!formStatus?.isDisabled}
-            canReset={formStatus?.isDirty || isIssuedAsNft || false}
-            onPrimaryButtonClick={formStatus?.triggerFormSubmit}
-            onSecondaryButtonClick={onSecondaryButtonClick}
-            onResize={setActionBarHeight}
-          />
-        </Container>
-      </CSSTransition>
-    </>
+    <BottomDrawer
```

Comment:
Good catch! I've added a `fixedScrollbar` prop to `BottomDrawer` that will set `overflow-y: scroll`. Did that because we shouldn't have the scrollbar there always since some of the drawer contents will not need scrolling